Userland/Shell/Tests/heredocs.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/sh
 
 source $(dirname "$0")/test-commons.inc
+# go to a writable directory
+cd /tmp
```

Comment:
Please:

- Put this in a separate commit
- Explain in the commit message why this is necessary

Reply:
Are you sure the commit linter isn't going to complain about having a separate commit? It was the reason I squashed in the first place.

Reply:
No, as long as all commits follow the commit message rules, there won't be a complaint. Plenty of PRs have multiple commits :)

Reply:
Ah OK, I'm still getting used to serenity commit policies, it seemed to indicate to squash everything after reviews.